tests/caasp/stack_update.pm

```diff
@@ -64,7 +64,13 @@ sub run {
     assert_and_click 'velum-update-reboot';
 
     # Update all nodes - this part takes long time (~2 minutes per node)
-    assert_screen "velum-$nodes-nodes-outdated", 300;
+    assert_screen [qw(velum-$nodes-nodes-outdated velum-sorry)], 300;
+    if (match_has_tag 'velum-sorry') {
+        record_soft_failure('bnc#1074836 - delay caused due to Meltdown');
+        # workaround for meltdown
+        send_key_until_needlematch 'velum-$nodes-nodes-outdated', 'f5', 10, 120;
```

Comment:
I believe you need double quotes around 'velum-$nodes-nodes-outdated', not singles

Reply:
True. I am restarting the test locally to see if it passes.